### PR TITLE
feat: update to new release 0.16.0

### PR DESCRIPTION
**Pull request by Pog3k**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,12 +25,14 @@ invoke changelog
 * Change auto changelog tool ([#30](https://github.com/eclipse/kiso-testing/issues/30))
 #### Continous integration
 
+* add skip lock for pipenv to prevent hangup
 * github-action for codecov ([#27](https://github.com/eclipse/kiso-testing/issues/27))
 #### Others
 
+* add skip lock for pipenv to prevent hangup
 * github-action for codecov ([#27](https://github.com/eclipse/kiso-testing/issues/27))
```
Comment (sebastianpfischer) [inline CHANGELOG.md:33]: This feels like a duplicate from above ^^

**Review by sebastianpfischer — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Pog3k) [reply CHANGELOG.md:33]: yup, 
The file is managed entirely by auto-changelog.

Makes no sense to cleanup this file manualy.
Maybe this stuff will be fixt in future releases.

**Review by BKaDamien — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (sebastianpfischer) [reply CHANGELOG.md:33]: Can we not find out why?

**Review by sebclrsn — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Pog3k) [reply CHANGELOG.md:33]: Suggestion would be to do it in a different task.
Its the python auto-changelog tool which then needs to be debugged.

The file will always be complete overwritten so we don't have to fix it in this task. 

IMO the release is more important.

**Review by sebastianpfischer — APPROVED** [review]

**Pull request merged**